### ISSUE TITLE
Flag for multiple args nests array inside array

### ISSUE THREAD

**Issue by ThatGerber** [docopt/docopt.rb#25]
Trying to write a script that takes multiple flags in a couple places, as well as multiple arguments.

```
Usage:
  script [options] [--ruleset FILE]... [RULE]...
  script [options] [--skip-rules CODE] [RULE]...

Command Options:
  --repository NAME       Name of repository to run rules against.
  -r, --ruleset FILE      Include ruleset found in FILE.
  -s, --skip-rules CODES  Comma separated list of rules to be skipped.
```

However, when I run  
`script --ruleset file/008.rb --ruleset file/009.rb --ruleset file/010.rb --skip-rules 30,40,50`  
it produces the following result:

```
--ruleset"=>
  [["file/008.rb",
    "file/009.rb",
    "file/010.rb"]],
 "--skip-rules"=>[["30,40,50"]],
```

I don't see a reason why those arrays would be nested, but it doesn't make sense to me and seems like a bug.

**Comment by ryanartecona** [comment]
@ThatGerber This indeed looks like a bug. I suspect #26 may fix it.

**Comment by alexspeller** [comment]
Looks like this should be fixed by `0.6.0` on rubygems
